### PR TITLE
Add cleanup tool to admin interface

### PR DESCRIPTION
**Pull request by annda**
This PR adds a simple db cleaner to the admin interface. It lists all invalid taggings (= tagged page does not exist) and lets the admin remove them all by clicking a button.

### CODE REVIEW

File: admin.php [hunk 1]
```diff
@@ -26,6 +26,11 @@ function forAdminOnly() {
      * Handle tag actions
      */
     function handle() {
+
+        if (!empty($_REQUEST['cmd']['clean'])) {
```
Comment (splitbrain) [inline admin.php]: I think this should check the securityToken to avoid CSRF attacks triggering the deletions of the tags

File: helper.php [hunk 2]
```diff
@@ -805,6 +821,42 @@ public function html_table() {
...
+
+        $html .= '<div class="table"><table class="inline plugin_tagging">';
+        $html .= '<thead><tr><th>' .
+            $this->getLang('admin nonexistent page') .
+            '</th><th>' .
+            $this->getLang('admin tags') .
+            '</th></tr></thead><tbody>';
+
+        foreach ($invalid as $row) {
+            $html .= '<tr><td>' . $row['pid'] . '</td><td>' . $row['tags'] . '</td></tr>';
```
Comment (splitbrain) [inline helper.php]: do we need to html escape something here?

File: admin.php (hunk 1 above)
Comment (annda) [reply admin.php]: fixed in a88577ddfa97b6eb7296192d1d65f0dd6ccc58e7

File: helper.php (hunk 2 above)
Comment (annda) [reply helper.php]: No, we have only cleaned tags and page ids here.

**Pull request merged**